src/libecalc/presentation/exporter/domain/exportable.py

```diff
@@ -0,0 +1,74 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Iterable, Iterator, List, Optional, Tuple
+
+from libecalc.common.units import Unit
+
+
+class ExportableType(str, Enum):
+    INSTALLATION = "INSTALLATION"
+
+
+class ConsumptionType(str, Enum):
+    FUEL = "FUEL"
+    ELECTRICITY = "ELECTRICITY"
+
+
+@dataclass
+class AttributeMeta:
```

Comment:
not sure if I agree on the naming here, a bit confusing for me (for now at least) - introducing another keyword ... but maybe I just need to get used to it ...

Reply:
`Attribute` or `Meta`? I'm open to proposals..it was named before I knew what to put in there. Currently it's just the necessary categories used to filter attributes in the queries.

Reply:
Both. If I had better names I would have mentioned it ;)